README.md

```diff
@@ -1 +1,24 @@
-# pubgolf
\ No newline at end of file
+# pubgolf
+
+## Setup Background Services (for Frontend Dev)
+
+Install the following:
+* Docker
+* Docker Compose
+
+Get a `.env` file (`cp .env.example .env`).
+
+Start background services:
+
+```
+docker-compose up --build -d
+```
```

Comment:
Can you add the command to see the logs for these to these once they're running?